test/e2e/vre-ch31-helpers.js

```diff
@@ -0,0 +1,108 @@
+const mock = require('./mock-helpers');
+
+function initApplicationSubmitMock() {
+  mock(null, {
+    path: 'v0/vre/okay',
+    verb: 'post',
+    value: {
+      data: {
+        attributes: {
+          guid: '123fake-submission-id-567'
+        }
+      }
+    }
+  });
+}
+
+function initDocumentUploadMock() {
+  mock(null, {
+    path: '/v0/vic/supporting_documentation_attachments',
+    verb: 'post',
+    value: {
+      data: {
+        attributes: {
+          guid: '123fake-submission-id-567'
+        }
+      }
+    }
+  });
+}
+
+function completeContactInformation(client, data) {
+  client
+    .fill('input[name="root_daytimePhone"]', data.daytimePhone)
+    .fill('input[name="root_eveningPhone"]', data.eveningPhone)
+    .fill('input[name="root_email"]', data.email)
+    .fill('input[name="root_view:confirmEmail"]', data.email);
+}
+
+function completeDisabilityInformation(client, data) {
+
+  client
+    .selectDropdown('root_disabilityRating', data.disabilityRating)
+    .fill('input[name=root_disabilities]', data.disabilities)
+    .fill('input[name=root_vaRecordsOffice]', data.vaRecordsOffice)
+    .selectRadio('root_view:inHospital', data['view:inHospital'])
+    .fill('input[name=root_view\\:hospital_hospitalName]', data.hospitalName)
```

Comment:
`\\` 🤷‍♂️

Reply:
We usually don't need that in e2e tests. Maybe by putting " around the attribute value?